src/theme.css

```diff
@@ -63,8 +63,10 @@
   --status-pending: var(--color-warning-secondary);
   --status-pending-100: var(--color-warning-secondary-100);
 
-  /* Main */
-  --main-surface: var(--color-white);
+  /* Surface colors */
+  --surface-main: var(--color-white);
+  --surface-current: var(--color-white);
```

Comment:
Moeten we dan ook iets met --text-color-main en --text-color-current?

Reply:
Nee, bij text kleuren zie je niet zoiets als terugkeren naar een beginstand. Bij surfaces zie je dat wel voorkomen. Elke volgende surface is een zebra-style van de vorige. Bijv. wit-kleur-wit.